pkg/rke/k8s_rke_system_images.go

```diff
@@ -9975,6 +10059,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 		},
 		// Enabled out of band post v2.8.2
+		"v1.27.10-rancher1-2": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.5.10",
+			Kubernetes:                "rancher/hyperkube:v1.27.10-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.96",
+			NginxProxy:                "rancher/rke-tools:v0.1.96",
+			CertDownloader:            "rancher/rke-tools:v0.1.96",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.96",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.22.28",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.22.28",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.22.28",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.6",
+			Flannel:                   "rancher/mirrored-flannel-flannel:v0.21.4",
+			FlannelCNI:                "rancher/flannel-cni:v0.3.0-rancher8",
+			CalicoNode:                "rancher/mirrored-calico-node:v3.26.3",
+			CalicoCNI:                 "rancher/calico-cni:v3.26.3-rancher1",
+			CalicoControllers:         "rancher/mirrored-calico-kube-controllers:v3.26.3",
+			CalicoCtl:                 "rancher/mirrored-calico-ctl:v3.26.3",
+			CalicoFlexVol:             "rancher/mirrored-calico-pod2daemon-flexvol:v3.26.3",
+			CanalNode:                 "rancher/mirrored-calico-node:v3.26.3",
+			CanalCNI:                  "rancher/calico-cni:v3.26.3-rancher1",
+			CanalControllers:          "rancher/mirrored-calico-kube-controllers:v3.26.3",
+			CanalFlannel:              "rancher/mirrored-flannel-flannel:v0.21.4",
+			CanalFlexVol:              "rancher/mirrored-calico-pod2daemon-flexvol:v3.26.3",
+			WeaveNode:                 "weaveworks/weave-kube:2.8.1",
+			WeaveCNI:                  "weaveworks/weave-npc:2.8.1",
+			AciCniDeployContainer:     "noiro/cnideploy:6.0.3.3.81c2369",
+			AciHostContainer:          "noiro/aci-containers-host:6.0.3.3.81c2369",
+			AciOpflexContainer:        "noiro/opflex:6.0.3.3.81c2369",
+			AciMcastContainer:         "noiro/opflex:6.0.3.3.81c2369",
+			AciOpenvSwitchContainer:   "noiro/openvswitch:6.0.3.3.81c2369",
+			AciControllerContainer:    "noiro/aci-containers-controller:6.0.3.3.81c2369",
+			PodInfraContainer:         "rancher/mirrored-pause:3.7",
+			Ingress:                   "rancher/nginx-ingress-controller:nginx-1.9.4-rancher1",
+			IngressBackend:            "rancher/mirrored-nginx-ingress-controller-defaultbackend:1.5-rancher1",
+			IngressWebhook:            "rancher/mirrored-ingress-nginx-kube-webhook-certgen:v20231011-8b53cabe0",
+			MetricsServer:             "rancher/mirrored-metrics-server:v0.6.3",
+			CoreDNS:                   "rancher/mirrored-coredns-coredns:1.10.1",
+			CoreDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.6",
+			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
+			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
+		},
+		// Enabled out of band post v2.8.3
```

Comment:
Same as another comment
```suggestion
		// Enabled out of band post v2.8.2
```

Reply:
addressed